### DISCUSSION TITLE
重复扣款，申请退款。

### DISCUSSION THREAD

**Discussion by KKLater** [getlantern/lantern-forum-cn#1071] [General]
您好！
使用老版本的App购买专业版时，多次提交购买失败，重试多次。最后邮箱收到扣款提醒，扣款两次。麻烦退款一次。
![1](https://github.com/user-attachments/assets/3ed43838-70a8-455e-b391-b0181cf926b6)
![2](https://github.com/user-attachments/assets/b70f905a-b117-4c39-9847-9385249bdb55)

另外邮箱未收到激活码，拜托给确认下。
谢谢！

**Comment by MaxMKMax** [comment]
@KKLater 您好. 可以的, 我帮您提交问题上去处理. 谢谢您支持蓝灯!

**Reply by KKLater** [reply]
@Mike777Khoo 麻烦先给一下激活码。目前没有激活码，免费流量也没有，只能用手机流量来查看信息。并且，自您回复消息，4天过去了，仍然没有收到退款处理。麻烦跟进。

**Comment by MaxMKMax** [comment]
@KKLater 您好. 我帮您查看一下. 一般退款处理需要 2-3 周. 您可以提供我一下账号邮箱吗? 我再次帮您提交问题.

**Reply by KKLater** [reply]
lshxin89@126.com
退款，我可以等2～3周。但是激活码给发一下呀。购买一周了，激活码一直没收到。谢谢。

**Reply by KKLater** [reply]
@Mike777Khoo 激活码还没收到，给处理解决下呀。

**Reply by MaxMKMax** [reply]
@KKLater 您好, 我再次帮您提交问题了. 今晚开会的话我再帮您问一下. 谢谢!

**Reply by KKLater** [reply]
@Mike777Khoo 没有收到激活码。没有收到退款～～～～～

**Comment by MaxMKMax** [comment]
@KKLater 您好. 非常抱歉啊. 美国团队今天才假期结束回来上线. 我再帮您发多一次问题请求. 非常抱歉让您久等了.
